app/assets/javascripts/editor/editor.js.erb

```diff
@@ -828,6 +840,28 @@ var CodeOceanEditor = {
         });
     },
 
+    showScoringFailureMessage: function () {
+        $.flash.warning({
+            icon: ['fa-solid', 'fa-triangle-exclamation'],
+            text: I18n.t('exercises.submit.failure')
```

Comment:
What do you think: Should we extend this message a little to inform the user that the scoring itself was performed successfully but "just" the score has not been transmitted successfully (and that they should retry later again)?